kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -0,0 +1,551 @@
+define([
```

Comment:
most of this was pulled directly from the appCell -> appParamsWidget. the main changes i made were:  
- remove references to batch mode
- filter out file input params and output fields (starting around line 420)
- remove all other file input references 

i left the appParamWidget in place for now, maybe once we extract the file inputs we can refactor that, but the module itself is so overloaded it's hard to say. let me know what y'all think

Reply:
does the location of this file make sense? or should it be under common/widgets? or somewhere else?

Reply:
I think the location makes sense. I believe that is where we have been putting all of the code we are extracting/making reusable.